src/banner.js

```diff
@@ -16,17 +16,19 @@ export function showCookieConfirmation() {
  */
 export function insertCookieBanner(onAccept) {
   // add a css block to the inserted html
-  const html = `${bannerHtml} <style>${bannerCss.toString()}</style>`;
-  document.getElementsByTagName('body')[0].innerHTML = html + document.getElementsByTagName('body')[0].innerHTML;
+  if (!process.env.NO_BANNER) {
```

Comment:
`process.env.NO_BANNER` is a string here. You will need to convert these strings to proper values.

`const NO_BANNER = process.env.NO_BANNER === "true"` or whatever

Reply:
While you're at it, you could save a level of indentation by doing
```
if(...) {
    return false;
}
```

Before the rest of the function

Reply:
I don't think it is.

`process.env.NO_BANNER === 'true' ;`
is false
`process.env.NO_BANNER === true ;`
is true

stringify converts from a javascript object to a string OR a boolean or number etc.

So it looks like you'd have to do the opposite if you wanted a string "true"